docs/manual/developer/03_creating_content.md

```diff
@@ -355,3 +355,372 @@ A profile should define these attributes:
+      and to use the targeted policy.
+```
+
+### Controls file format
+
+This is a complete schema of the YAML file format.
+
+```
+id: policy ID (required key)
+title: short title (required key)
+source: a link to the original policy, eg. a URL of a PDF document
+levels: a list of levels, sorted from lowest to highest
+controls: a list of controls (required key)
+  - id: control ID (required key)
+    title: control title
+    description: description of the control in a few sentences
+    automated: Can be one of: ["yes", "no", "partially"]. Default value: "yes".
+    level: The policy level that the control belongs to.
+    notes: a short paragraph of text
+    rules: a list of rule IDs that cover this control
+    related_rules: a list of related rules
+    note: a short paragraph of text
+    controls: a nested list of controls
+```
+
+Full example of a controls file:
+
+```
+id: abcd
+title: ABCD Benchmark for securing Linux systems
+source: https://www.abcd.com/linux.pdf
+levels:
+  - low
+  - high
+controls:
+  - id: R1
+    level: low
+    title: User session timeout
+    description: >-
+      Remote user sessions must be closed after a certain
+      period of inactivity.
+    automated: yes
+    rules:
+    - sshd_set_idle_timeout
+    - accounts_tmout
+    - var_accounts_tmout=10_min
+    - configure_crypto_policy
+    notes: >-
+      Certain period of inactivity is vague.
+  - id: R2
+    title: Minimization of configuration
+    description: >-
+      The features configured at the level of launched services
+      should be limited to the strict minimum.
+    automated: no
+    note: >-
+      This is individual depending on the system workload
+      therefore needs to be audited manually.
+    related_rules:
+       - systemd_target_multi_user
+  - id: R3
+    title: Enabling SELinux targeted Policy
+    description: >-
+      It is recommended to enable SELinux in enforcing mode
+      and to use the targeted policy.
+    automated: yes
+    rules:
+      - selinux_state
+  - id: R4
+    title: Configure authentication
+    description: >-
+      Ensure authentication methods are functional to prevent
+      unauthorized access to the system.
+    controls:
+      - id: R4.a
+        title: Disable administrator accounts
+        automated: yes
+        level: low
+        rules:
+          -  accounts_passwords_pam_faillock_deny_root
+      - id: R4.b
+        title: Enforce password quality standards
+        automated: yes
+        level: high
+        rules:
+          - accounts_password_pam_minlen
+          - accounts_password_pam_ocredit
+          - var_password_pam_ocredit=1
+```
+
+### Using controls in profiles
+
+Later, we can use the policy requirements in profile YAML. Let’s say that we
+will define a “Desktop” profile built from the controls.
+
+To use controls, we add them under `selection` keys in the profile. The entry
+has the form `policy_id:control_id`.
```

Comment:
Is this right? Looking at the ANSSI profiles, I think the form is:

```
policy_id:control_id[:level_id]
```

Where `level_id` is optional. 


(You state it here like it is the definitive form, but below you introduce another form -- I figured it makes sense to clear it up.)